ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -85,7 +85,11 @@ public String getVersion() {
             currentSensitivityAnalysisResult = runSystematicSensitivityAnalysisWithParametersSwitch(network, crac, computationManager, linearRaoParameters, useFallbackSensiParams);
         } catch (SensitivityComputationException e) { // Failure if some sensitivities are not computed
             resultVariantManager.deleteVariants(preOptimVariant, bestResultVariant);
-            return CompletableFuture.completedFuture(new RaoResult(RaoResult.Status.FAILURE));
+            LinearRaoResult resultExtension = new LinearRaoResult();
+            resultExtension.setSystematicSensitivityAnalysisParameters(useFallbackSensiParams.get());
```

Comment:
and here we could then return a SystematicSensitivityAnalysisParameters.FAILURE, to say neither of both (or one if there is no fallback) configs worked.